Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs

```diff
@@ -53,7 +55,7 @@ internal static class ImageViewExtensions
 							animation = await animationHandler.LoadImageAnimationAsync(newImageSource, imageView.Context);
 					}
 
-					if(animation == null)
+					if (animation == null)
```

Comment:
We should still set the `ErrorSource` for an animation if the animation fails to load yes?

Reply:
I can't reproduce this... Do you know how can I check if the animation fails?